.aegir.js

```diff
@@ -38,12 +54,7 @@ const before = (done) => {
 }
 
 const after = (done) => {
-  setTimeout(() => node.stop((err) => {
-    if (err) {
-      return done(err)
-    }
-    server.stop(done)
-  }), 2000)
+  setTimeout(() => require('async/each')([node, server, server2], (s, n) => s.stop(n), done), 2000)
```

Comment:
a) Please do not require modules throughout the codebase, always require it at the top.
b) Preserve the `{}` when nesting multiple calls.
c) Use async/parallel for this